### ISSUE TITLE
Missing commands in Oracle Linux

### ISSUE THREAD

**Issue by janihur** [OraOpenSource/OXAR#147]
```bash
[root@localhost vagrant-deploy]# ./build.sh

* Loading configurations *

######################################################################## 100.0%
/tmp/vagrant-deploy/utils/ords_version.sh: line 28: unzip: command not found
/tmp/vagrant-deploy/utils/ords_version.sh: line 35: java: command not found
The version of ORDS you are attempting to install is not supported
Please grab the latest version and try again
[root@localhost vagrant-deploy]#
```

**Comment by tschf** [comment]
@fuzziebrain This looks as a result of my adding an ORDS version validation, by getting the version - which I suppose occurs before all the required packages are fetched.

**Comment by martindsouza** [comment]
@tschf As discussed a simple solution would be to do a custom `yum` ahead of this installing `java` and `unzip`.  Or reorder process.

**Comment by martindsouza** [comment]
Resolved in #149
